tests/chia_log/parsers/test_wallet_added_coin_parser.py

```diff
@@ -0,0 +1,27 @@
+# std
+import unittest
+from pathlib import Path
+
+# project
+from src.chia_log.parsers.wallet_added_coin_parser import WalletAddedCoinParser
+
+
+class TestWalletAddedCoinParser(unittest.TestCase):
+    def setUp(self) -> None:
+        self.parser = WalletAddedCoinParser()
+        self.example_logs_path = Path(__file__).resolve().parents[1] / "logs/wallet_added_coin"
+        with open(self.example_logs_path / "nominal.txt") as f:
```

Comment:
Encoding needs to be specifically set to UTF-8 or unit tests will fail under Windows